test/lhs_tests.js

```diff
@@ -51,4 +51,28 @@ describe('LHS', function () {
         const best = facts.get('best');
         assert.equal(best.length, 1);
     });
+
+    it('testing when() on type()', function () {
```

Comment:
This test doesn't discriminate between when() doing its job correctly and it simply returning everything. I recommend having 2 paragraphs of different IDs and checking to make sure it grabs the right one.

And then, of course, there is no need for the max() rule of the score(5).